lib/alephant/broker/cache/cached_object.rb

```diff
@@ -0,0 +1,71 @@
+require 'alephant/logger'
+require 'time'
+
+module Alephant
+  module Broker
+    module Cache
+      class CachedObject
+        include Logger
+        attr_reader :s3_obj
+
+        DEFAULT_TTL = 10
+
+        def initialize(obj)
+          logger.info(event:   'SettingCachedObject',
+                      content: obj,
+                      method:  "#{self.class}#initialize")
+
+          @s3_obj = obj
+        end
+
+        def update(obj)
+          logger.info(event:       'UpdatingCachedObject',
+                      old_content: @s3_obj,
```

Comment:
It's the whole object - `update` isn't actually called in the code, I just kept it there from the original commits as it could be useful in the future...